mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/25 - component view

Closes #25 

To test:
- Run Introductory scenario
- Follow scenario steps up to opening the Attack Log
- Click Component tab to open view
- Select either "Blato Desktop Select" computer from the list
- Enable "Beware of Email Attachments" and "No External Software"
- Repeat with other computer
- Make sure objectives get completed correctly

Notes:
- There is still a TON to do on the ComponentView, but I will make separate cases for each individual aspect of the UI so we can break them down to doable sized tasks
